code/modules/mob/living/silicon/robot/drone/drone.dm

```diff
@@ -280,20 +280,6 @@
 				return
 	..()
 
-/mob/living/silicon/robot/drone/Bump(var/atom/movable/AM, yes)
```

Comment:
As pointed out to me, it's actually an issue that stop drones being able to do some of the things they need to do, like interacting with specific machines (Charging stations for example)